content/config/otel.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Open Telemetry for the Docker CLI
+description: Learn about how you can leverage the Docker's capabilities to capture OTel traces
+keywords: otel, telemetry, traces, metrics, logs
+---
+
+{{< introduced engine 26.1.0 >}}
+
+The Docker CLI supports [OTel](https://opentelemetry.io/docs/) instrumentation
+for emitting metrics about command invocations. This is disabled by default.
+You can configure the CLI to start emitting metrics to the endpoint that you
+specify. This allows you to capture information about your `docker` command
+invocations for more insight into your Docker usage.
+
+The destination of these metrics is completely opt-in, and you control the
```

Comment:
nit: destination is opt-in? maybe we should use something like `exporting` or `creation`?

Reply:
Agreed. Maybe it would be clearer to say that `exporting is opt-in, and can be configured/controled...`